### PR TITLE
SASS-4100 - Update README.md file for State Benefits FE

### PR DESCRIPTION
**Pull request by satyaayinala**
### Description

Update README.md file for State Benefits FE

Add a link to the relevant story in Jira
[SASS-4100](https://jira.tools.tax.service.gov.uk/browse/SASS-4100)

##### Before Reviewing

- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [x]  Have you checked to ensure all dependencies are up to date?
- [x]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing

- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing

- [x]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request?
  i.e. does it need another rebase?
- [x]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [x]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you run the smoke tests? (where applicable)
- [ ]  Have you run the performance tests? (where applicable)
- [x]  Have you addressed warnings where appropriate?
- [x]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised

- [x]  Have you checked the PR Builder passes?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,57 @@
...
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+
+To run the service locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+    sm --stop INCOME_TAX_STATE_BENEFITS_FRONTEND
+    ./run.sh **OR** sbt -Dplay.http.router=testOnlyDoNotUseInAppConf.Routes run
+
+This service runs on port: `localhost:9376`
```
Comment (georgievh) [inline README.md:27]: Please move this after the line where Income Tax State Benefits FE service is started to make it clear that the port is for that service, not for the other ones

File: README.md [hunk 2]
```diff
@@ -1,7 +1,57 @@
...
+
+### Feature Switches
+
+| Feature        | Environments Enabled In            |
+|----------------|------------------------------------|
+| Encryption     | None                               |
```
Comment (georgievh) [inline README.md:33]: You can remove Encryption from here

**Review by georgievh — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (satyaayinala) [reply README.md:27]: Moved

File: README.md (hunk 2 above)
Comment (satyaayinala) [reply README.md:33]: Removed

File: README.md [hunk 3]
```diff
@@ -1,7 +1,57 @@
...
+| Welsh Language | Local, QA, Staging, ET, Production |
+
+### State Benefits Sources (HMRC-Held and Customer Data)
+State Benefits data can come from different sources: HMRC-Held and Customer. HMRC-Held data is state benefits data that HMRC have for the user within the tax year, prior to any updates made by the user. The state benefits data displayed in-year is HMRC-Held.
+
+Customer data is provided by the user. At the end of the tax year, users can view any existing state benefits data and make changes (create, update and delete).
+
+Examples can be found here in the [income-tax-submission-stub](https://github.com/hmrc/income-tax-submission-stub/blob/main/app/models/StateBenefitsUsers.scala)
```
[the remark this answers is not in the document]
Comment (satyaayinala) [reply README.md:41]: Changed text

**Review by georgievh — APPROVED** [review]

**Pull request merged**